README.md

```diff
@@ -52,5 +52,5 @@
 # GitHub Stats
 
 <p align="center">
-    <img width="500" src="../github-metrics.svg" alt="GitHub Stats">
+    <img width="500" src="github-metrics.svg" alt="GitHub Stats">
```

Comment:
the image is failing to load

Reply:
also main readme is under `profile/readme.md`

Reply:
> the image is failing to load

locally it was loading. I will update it